samples/matter/lock/sysbuild/ipc_radio/boards/nrf7002dk_nrf5340_cpuapp nrf7001.conf

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
Oh, I spotted one thing, there is a space in the file name :)

Reply:
Oh, that's embarrassing, correcting it now.